### ISSUE TITLE
`segment.bundles` package not being tested

### ISSUE THREAD

**Issue by jhlegarreta** [nipy/dipy#1889]
## Description
The [`bundles`](https://github.com/nipy/dipy/blob/master/dipy/segment/bundles.py) package's [unit test](https://github.com/nipy/dipy/blob/master/dipy/segment/tests/test_bundles.py) is empty.

The test file's history shows that there has been some refactoring at some point, but it looks like some testing code was either misplaced or the `test_bundles.py` needs to be fed with the appropriate testing code.

codecov [reports](https://codecov.io/gh/nipy/dipy/src/master/dipy/segment/bundles.py) a 91.47% coverage for that package, but it may well be because it is being called as an intermediate step from other tests or code fragments.

**Comment by arokem** [comment]
Looks like prior to [this](https://github.com/nipy/dipy/commit/b3a6d0eb376ca2211ff514e17cc71fbd812804fc#diff-ca53b3e9e0ca17697e9c4822526980a8) commit there wasn't a test file for this module. Then, I moved the tests that I had created in [this](https://github.com/nipy/dipy/commit/e255facb10c28096016746649d2dba378230126c#diff-ca53b3e9e0ca17697e9c4822526980a8) commit. 

Maybe we should simply remove this file altogether? Looks like most of that stuff is tested in  `test_rb.py`. Another option would be to rename that to `test_bundles.py`  for clarity. WDYT @Garyfallidis?
